### PR TITLE
Pass method call directly to inner value

### PR DESCRIPTION
**Pull request by rap1ds**

### CODE REVIEW

File: spec/spec.rb [hunk 1]
```diff
@@ -156,4 +156,12 @@ def test_case_when(case_value, match_value, non_match_value)
       expect(Some([1, 2, 3]).map { |arr| arr.map { |v| v * v } }.get).to eql([1, 4, 9])
     end
   end
+
+  describe "inner" do
+    it "forwards all (also Enumerable) methods" do
+      expect(Some(["first", "second", "third"]).inner.first.upcase.or_else { false }).to eql("FIRST")
+      expect(Some(["first", "second", "third"]).inner.map(&:upcase).or_else { false }).to eql(["FIRST", "SECOND", "THIRD"])
+      expect(Some([]).inner.first.upcase.or_else { "NONE" }).to eql("NONE")
```
Comment (rap1ds) [inline spec/spec.rb]: Add test for None (it will probably break)
